spec/controllers/pages_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe PagesController do
+  describe 'GET #search' do
+    it 'renders the search page' do
+      get :search
+      expect(response.status).to eq(200)
```

Comment:
Make sure your test and the message are in line.  In this case the test is *actually* testing if the page returns successful.  To test that it renders the search page we'd say something like `expect(response).to render_template(:search)`.